src/app/lib/README.md

```diff
@@ -41,3 +41,92 @@ Here are the font sizes in `px`s:
 | Long Primer  | 15            | 18              | 15            | 18              | 14            | 18              | Image caption & Video caption & Onward journey timestamps        |
 | Brevier      | 14            | 16              | 14            | 18              | 13            | 16              | Article timestamps                                               |
 | Minion       | 12            | 16              | 12            | 16              | 12            | 16              | Article category                                                 |
+
+# Layout
+
+## UX grid definition
+
+As per the [GEL grid Guidelines](https://www.bbc.co.uk/gel/guidelines/grid) we have the following UX grid definition:
+
+| Breakpoints     | Gutters | Margins | Grid width limitations |
+| :-------------- | :------ | :------ | :--------------------- |
+| 0px - 399px     | 8px     | 8px     | dynamic / full width   |
+| 400px - 599px   | 8px     | 16px    | dynamic / full width   |
+| 600px - 1007px  | 16px    | 16px    | dynamic / full width   |
+| 1008px - 1280px | 16px    | 16px    | static / 1008px        |
+| 1280px +        | 16px    | 16px    | static / 1280px        |
+
+## Article layout requirements
+
+In the article project we have requirements for items to span wider than the grid, these "full bleed" items will span the full width of the viewport effectively breaking out of the grid.
+
+The article layout UX design has the following requirements:
+
+| Breakpoints     | Columns | Content* spans X columns | 
+| :-------------- | :------ | :----------------------- | 
+| 0px - 599px     | 6       | 6 of 6                   | 
+| 600px - 1007px  | 6       | 4 of 6                   | 
+| 1008px - 1280px | 8       | 6 of 8                   | 
+| 1280px +        | 10      | 6 of 10                  | 
+* content contained within the grid, therefore not "full bleed"
```

Comment:
```suggestion
\* content contained within the grid, therefore not "full bleed"
```

Escape the `*` to have it render as an asterisk.

Reply:
Hmm this doesn't appear to have gone through! (Might be a bug with the new GitHub 'suggest' feature)

Reply:
Oops - was just looking at the wrong commit